src/pages/home/report/ReportActionItemCreated.tsx

```diff
@@ -98,33 +88,4 @@ function ReportActionItemCreated({report, personalDetails, policy, reportID}: Re
 
 ReportActionItemCreated.displayName = 'ReportActionItemCreated';
 
-export default withOnyx<ReportActionItemCreatedProps, ReportActionItemCreatedOnyxProps>({
-    report: {
-        key: ({reportID}) => `${ONYXKEYS.COLLECTION.REPORT}${reportID}`,
-    },
-
-    policy: {
-        key: ({policyID}) => `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
-    },
-
-    personalDetails: {
-        key: ONYXKEYS.PERSONAL_DETAILS_LIST,
-    },
-})(
-    memo(
-        ReportActionItemCreated,
-        (prevProps, nextProps) =>
-            prevProps.policy?.name === nextProps.policy?.name &&
-            prevProps.policy?.avatarURL === nextProps.policy?.avatarURL &&
-            prevProps.report?.stateNum === nextProps.report?.stateNum &&
-            prevProps.report?.statusNum === nextProps.report?.statusNum &&
-            prevProps.report?.lastReadTime === nextProps.report?.lastReadTime &&
-            prevProps.report?.description === nextProps.report?.description &&
-            prevProps.personalDetails === nextProps.personalDetails &&
-            prevProps.policy?.description === nextProps.policy?.description &&
-            prevProps.report?.reportName === nextProps.report?.reportName &&
-            prevProps.report?.avatarUrl === nextProps.report?.avatarUrl &&
-            lodashIsEqual(prevProps.report?.invoiceReceiver, nextProps.report?.invoiceReceiver) &&
-            prevProps.report?.errorFields === nextProps.report?.errorFields,
```

Comment:
NAB: I assume it's ok to remove these conditions because the React Compiler automatically memoizes now?

Reply:
I think react compiler only memoize like what useMemo did, for example, it will memorize
```
const shouldDisableDetailPage = ReportUtils.shouldDisableDetailPage(report);
```

but not the component itself.

Reply:
Those props coming from withOnyx, but since we replace it with useOnyx, I remove them. Do you think we should keep using withOnyx in this case?

Reply:
Oh right, thanks for explaining. I hadn't made that connection. All good.